test/specs/errors/helpers/http_error.js

```diff
@@ -1,68 +1,69 @@
 var nock = require('nock');
 var server, ermRest, ermrestUrl, ops = {allowUnmocked: true};
 
-var errorCodes = { 
-	"400" : { type: "BadRequestError" },
-	"401" : { type: "UnauthorizedError" },
-	"403" : { type: "ForbiddenError" },
-	"404" : { type: 'NotFoundError' }, 
-	"409" : { type: "ConflictError" },
-	"500" : { type: "InternalServerError" },
-	"503" : { type: "ServiceUnavailableError" }
+var errorCodes = {
+    "400" : { type: "BadRequestError" },
+    "401" : { type: "UnauthorizedError" },
+    "403" : { type: "ForbiddenError" },
+    "404" : { type: 'NotFoundError' },
+    "409" : { type: "ConflictError" },
+    "500" : { type: "InternalServerError" },
+    "503" : { type: "ServiceUnavailableError" }
 };
 
 exports.setup = function(options) {
-	server = options.server;
-	ermRest = options.ermRest;
-	ermrestUrl = options.url.replace('ermrest', '');
+    server = options.server;
+    ermRest = options.ermRest;
+    ermrestUrl = options.url.replace('ermrest', '');
 };
 
 exports.testForErrors = function(method, errorTypes, cb, message, mockUrl) {
```

Comment:
I feel like now that you've spend the time to figure out this function, it's worth adding few comments explaining it. So we don't have to go through the same process again.